src/orchard/inspect.clj

```diff
@@ -82,7 +82,7 @@
   [inspector]
   (merge (reset-index inspector)
          {:value nil, :stack [], :path [], :pages-stack [],
-          :current-page 0, :rendered '(), :indentation 0}))
+          :current-page 0, :rendered [], :indentation 0}))
```

Comment:
We have existing patterns in this ns such as `(update-in [:rendered] concat (list expr))` so I'd rather not have this touched.

Reply:
I changed that as well. Concat is generally an antipattern. https://stuartsierra.com/2015/04/26/clojure-donts-concat

Reply:
Alright

Either way, any change that has the risk of going wrong deserves its own commit, if not its own PR.

PRs which aren't constrained to a single topic are harder to review.

Please don't make maintainers squint/head-scratch at changes - present them nicely so that it's all safe and streamlined.

Hope it helps for future occasions!

Reply:
Removed that commit from this PR.